Source/WebKit/UIProcess/WebsiteData/WebsiteDataStoreConfiguration.h

```diff
@@ -318,6 +323,9 @@ class WebsiteDataStoreConfiguration : public API::ObjectImpl<API::Object::Type::
 #if PLATFORM(COCOA)
     RetainPtr<CFDictionaryRef> m_proxyConfiguration;
 #endif
+#if USE(CURL)
+    String m_alternativeServicesStorageFile;
```

Comment:
This is redundant with m_alternativeServicesDirectory

Reply:
I will fix it.

Reply:
Fixed